lib/liquid/standardfilters.rb

```diff
@@ -391,6 +391,17 @@ def round(input, n = 0)
       raise Liquid::FloatDomainError, e.message
     end
 
+    # Defaults are passed as nil so systems can easily override
+    def format_number(input, options = {})
+      options = {} unless options.is_a?(Hash)
```

Comment:
I would raise an error if `options` is not a hash so that we don't silently skip over bugs in the Liquid template.